RealmWeaver/ModuleWeaver.cs

```diff
@@ -810,6 +809,83 @@ private static bool IsRealmList(PropertyDefinition property)
         return property.PropertyType.Name == "RealmList`1" && property.PropertyType.Namespace == "Realms";
     }
 
+    private static bool IsDateTimeOffset(PropertyDefinition property)
```

Comment:
It bugs me that we do string comparison in the first place - I'd much rather we compare `TypeDefinition` instances.